artemis/reporting/modules/nuclei/reporter.py

```diff
@@ -110,6 +122,15 @@ def _is_url_without_query_fragment(url: str) -> bool:
                 + Config.Modules.Nuclei.NUCLEI_TEMPLATES_TO_SKIP_WHEN_REPORTING
             ):
                 continue
+            if is_lfi_template(template):
```

Comment:
I went with that change, but I'm still thinking that maybe it's better to do change to file "nuclei_template_groups.json"
with
{
"dast/vulnerabilities/lfi/lfi-keyed.yaml" : "lfi"
"dast/vulnerabilities/lfi/linux-lfi-fuzz.yaml": "lfi"
"dast/vulnerabilities/lfi/windows-lfi-fuzz.yaml": "lfi"
}
and manually add more lfi if their appears.

We may also then add e.g. http/cves/2015/CVE-2015-4074.yaml - which is also lfi, but not dast and not currently catched by that check in code.